cmd/juju/status/health.go

```diff
@@ -0,0 +1,252 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package status
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"launchpad.net/gnuflag"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/juju/osenv"
+	"github.com/juju/juju/status"
+)
+
+type statusHealthCommand struct {
+	modelcmd.ModelCommandBase
+	out     cmd.Output
+	isoTime bool
+}
+
+var statusHealthDoc = `
+This command will report the health of the Juju environment
+The command outputs a single line summary with any unhealthy units 1 per line on following lines.
+The return code of this is 0 for Okay, 1 for warning, 2 for critical, 3 for unknown
+`
+
+// NewHealthCommand returns a command that reports the health of the Juju deployment
+func NewHealthCommand() cmd.Command {
+	return modelcmd.Wrap(&statusHealthCommand{})
+}
+
+func (c *statusHealthCommand) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:    "status-health",
+		Purpose: "output the health of the Juju environment",
+		Doc:     statusHealthDoc,
+		Aliases: []string{"health"},
+	}
+}
+
+func (c *statusHealthCommand) SetFlags(f *gnuflag.FlagSet) {
+	f.BoolVar(&c.isoTime, "utc", false, "display time as UTC in RFC3339 format")
+}
+
+func (c *statusHealthCommand) Init(args []string) error {
+	// If use of ISO time not specified on command line,
+	// check env var.
+	if !c.isoTime {
+		var err error
+		envVarValue := os.Getenv(osenv.JujuStatusIsoTimeEnvKey)
+		if envVarValue != "" {
+			if c.isoTime, err = strconv.ParseBool(envVarValue); err != nil {
+				return errors.Annotatef(err, "invalid %s env var, expected true|false", osenv.JujuStatusIsoTimeEnvKey)
+			}
+		}
+	}
+	return nil
+}
+
+// statusHealthCommand.Run output and return code should conform to this commonly used standard
+// https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/pluginapi.html
+func (c *statusHealthCommand) Run(ctx *cmd.Context) error {
+	var exitCode int
+	var notOkay []string
+
+	// First determine the status
+	apiclient, err := c.NewAPIClient()
+	if err != nil {
+		return errors.Trace(err)
+	}
+	defer apiclient.Close()
+
+	status, err := apiclient.Status([]string{})
+	if err != nil {
+		// problems getting status report Unknown
+		exitCode = 3
+		notOkay = append(notOkay, err.Error())
+	}
+
+	//Parse status setting exitCode and summary message to the most severe status
+	if status != nil {
+		sf := NewStatusFormatter(status, c.isoTime)
+		formatted := sf.format()
+		code, issues := formatted.Health()
+		if code > exitCode {
+			exitCode = code
+		}
+		notOkay = append(notOkay, issues...)
+	}
+
+	//set summary line based on most sever exitcode
+	summary := "Juju Health "
```

Comment:
why not just 

fmt.Fprintln(ctx.Stdout, "Juju Health", summary) 

where summary is just the case block above ?